executor/linux/service.go

```diff
@@ -81,7 +82,12 @@ func (c *client) PlanService(ctx context.Context, ctn *pipeline.Container) error
 func (c *client) ExecService(ctx context.Context, ctn *pipeline.Container) error {
 	b := c.build
 	r := c.repo
-	l := c.serviceLogs[ctn.ID]
+
+	result, ok := c.serviceLogs.Load(ctn.ID)
+	if !ok {
+		return fmt.Errorf("unable to get log from client")
```

Comment:
Thoughts on updating the error message for the resource here?

Maybe something like:

`unable to get service log from client`

Reply:
Yeah being a bit more verbose would definitely help the parsing when you're looking at the output in a log aggregator. Will update!